### PR TITLE
Translation of the "AJAX and APIs" page

### PR DESCRIPTION
**Pull request by lazaroalvarenga**
#1

### CODE REVIEW

File: content/docs/faq-ajax.md [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+### Onde eu devo fazer uma chamada AJAX no ciclo de vida do componente? {#onde-eu-devo-fazer-uma-chamada-ajax-no-ciclo-de-vida-do-componente}
 
-You should populate data with AJAX calls in the [`componentDidMount`](/docs/react-component.html#mounting) lifecycle method. This is so you can use `setState` to update your component when the data is retrieved.
+Você deve preencher dados com chamadas AJAX no método [`componentDidMount`](/docs/react-component.html#mounting) do ciclo de vida. Isto é para que você consiga usar `setState` para atualizar seu componente quando os dados forem recebidos.
 
-### Example: Using AJAX results to set local state {#example-using-ajax-results-to-set-local-state}
+### Exemplo: Usando resultados AJAX para definir o estado local {#exemplo-usando-ajax-para-definir-o-estado-local}
 
-The component below demonstrates how to make an AJAX call in `componentDidMount` to populate local component state. 
+O componente abaixo demonstra como deve fazer uma chamada AJAX no `componentDidMount` para preencher o estado local.
```
Comment (cezaraugusto) [inline content/docs/faq-ajax.md]: at the end could you please also add "state" in there?

```diff
-para preencher o estado local
+para preencher o estado (state) local
```

**Comment by glaucia86** [comment]
cc: @cezaraugusto @WendellAdriel can see this PR?! For me looks good to me! :)

**Comment by lazaroalvarenga** [comment]
@glaucia86 I updated with changes to standardize the translation of the word `call`. It's fine?

**Comment by WendellAdriel** [comment]
@glaucia86 @cezaraugusto @lazaroalvarenga 
I really think that we shouldn't translate code blocks, I put that as the first item on #4

File: content/docs/faq-ajax.md [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
 
-The example API returns a JSON object like this:
+A API de exemplo retorna um objeto JSON como este:
 
 ```
 {
-  "items": [
-    { "id": 1, "name": "Apples",  "price": "$2" },
-    { "id": 2, "name": "Peaches", "price": "$5" }
+  "itens": [
```
Comment (cezaraugusto) [inline content/docs/faq-ajax.md]: please do not translate code blocks unless it's a comment

File: content/docs/faq-ajax.md [hunk 3]
```diff
@@ -1,79 +1,79 @@
...
+      return <div>Carregando...</div>;
     } else {
       return (
         <ul>
-          {items.map(item => (
-            <li key={item.name}>
-              {item.name} {item.price}
+          {itens.map(item => (
+            <li key={item.nome}>
+              {item.nome} {item.preço}
```
Comment (cezaraugusto) [inline content/docs/faq-ajax.md]: ditto for all the above

**Comment by lazaroalvarenga** [comment]
> @glaucia86 @cezaraugusto @lazaroalvarenga
> I really think that we shouldn't translate code blocks, I put that as the first item on #4

ok! Sorry for that. I'll unmake the code blocks translation.

File: content/docs/faq-ajax.md (hunk 1 above)
Comment (lazaroalvarenga) [reply content/docs/faq-ajax.md]: done!

File: content/docs/faq-ajax.md (hunk 2 above)
Comment (lazaroalvarenga) [reply content/docs/faq-ajax.md]: Sure. done!

File: content/docs/faq-ajax.md (hunk 3 above)
Comment (lazaroalvarenga) [reply content/docs/faq-ajax.md]: done!

**Comment by glaucia86** [comment]
@lazaroalvarenga todos os acertos que vc for fazendo de acordo com review marque como 'Resolve Conversation' ;)

**Comment by glaucia86** [comment]
cc: @cezaraugusto @lazaroalvarenga already did the changes here. I think that we can approve his request. What do u think?

**Pull request merged**